lib/Kbuild.sub

```diff
@@ -61,4 +61,7 @@ m0tr_objects += lib/linux_kernel/finject_init.o \
                   lib/linux_kernel/semaphore.o \
                   lib/linux_kernel/timer.o \
                   lib/linux_kernel/kuuid.o \
-                  lib/linux_kernel/varr.o
+                  lib/linux_kernel/varr.o \
+                  lib/linux_kernel/linux_ccnt/aarch64_cycle_counter.o
```

Comment:
Drop `linux_ccnt/` directory, it's not needed here. Place the files at `linux_kernel/` folder.